Fix errors caught by travis ci

Hi please fix the errors caught by travis ci.

148Running "nodeunit:tests" (nodeunit) task

149Testing nodeunit_test.js.F....

150>> nodeunit - fail

151>> Message: Operator should display for multiline.

152>> Error: false == true

153>> at /home/travis/build/gruntjs/grunt-contrib-nodeunit/test/nodeunit_test.js:18:12

154>> at callDone (node_modules/grunt/node_modules/grunt-legacy-util/index.js:159:5)

155>> at ChildProcess.<anonymous> (node_modules/grunt/node_modules/grunt-legacy-util/index.js:198:5)

156>> at ChildProcess.emit (events.js:110:17)

157>> at maybeClose (child_process.js:1015:16)

158>> at Process.ChildProcess._handle.onexit (child_process.js:1087:5)

159

160Warning: 1/24 assertions failed (5848ms) Use --force to continue.

161

162Aborted due to warnings.

163npm ERR! Test failed.  See above for more details.

164

165The command "npm test" exited with 1.

Since this repo may be causing other repos to fail.

The 0.10 test is the only one that passes but 0.12 and 4 have been failing.

--- a/tasks/nodeunit.js
+++ b/tasks/nodeunit.js
@@ -40,8 +40,8 @@ module.exports = function(grunt) {
     e._message = e.message;
 
     // Pretty-formatted objects.
-    var actual = util.inspect(e.actual, false, 10, true);
-    var expected = util.inspect(e.expected, false, 10, true);
+    var actual = util.inspect(e.actual, { depth: 10, colors: true });
+    var expected = util.inspect(e.expected, { depth: 10, colors: true });
 
     var indent = function(str) {
       return ('' + str).split('\n').map(function(s) { return '  ' + s; }).join('\n');

--- a/test/fixtures/fail.js
+++ b/test/fixtures/fail.js
@@ -9,8 +9,8 @@ exports.fail = {
     var error = new Error('Something arbitrary');
     // Must be long enough that the inspect calls try to
     // wrap the line for indentation.
-    error.actual = { foo: 'bar', something: 'complex', more: 'more' };
-    error.expected = "No you didn't"
+    error.actual = { foo: 'bar', something: 'complex', more: 'more', even: 11335577992 };
+    error.expected = "No you didn't";
     error.showDiff = true;
     throw(error);
     test.done();

